soc/nordic/soc.yml

```diff
@@ -21,6 +21,14 @@ family:
               - name: cpunet
       - name: nrf54l
         socs:
+          - name: nrf54l05
+            cpuclusters:
+              - name: cpuapp
+              - name: cpuflpr
+          - name: nrf54l10
+            cpuclusters:
+              - name: cpuapp
+              - name: cpuflpr
```

Comment:
Down in this file there should be added elements for nrf54l05 and nrf54l10, the same as there is nrf54l15
(so that reset, erase and recovery will work correctly)